core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/dot-experiments-configuration.component.ts

```diff
@@ -131,4 +140,20 @@ export class DotExperimentsConfigurationComponent implements OnInit {
             }
         });
     }
+
+    /**
+     * Save the description of the experiment
+     * @param description
```

Comment:
types in the params ?

Reply:
This extension is really cool for comments https://marketplace.visualstudio.com/items?itemName=oouo-diogo-perdigao.docthis

Reply:
@zJaaal I use Webstorm :( 
@hmoreras added.